tests/test_preprints.py

```diff
@@ -2308,6 +2311,12 @@ def setUp(self):
         self.preprint = PreprintFactory(creator=self.user)
         self.JWE_KEY = jwe.kdf(settings.WATERBUTLER_JWE_SECRET.encode('utf-8'), settings.WATERBUTLER_JWE_SALT.encode('utf-8'))
 
+    # enable the ELASTICSEARCH_METRICS switch for all tests
```

Comment:
Does this need to be all tests in this class or can you use the`@override_switch(features.ELASTICSEARCH_METRICS, active=True)` decorator on the ones that do need it? Not sure which is more efficient.

Reply:
I just thought it was safer to make the test more comprehensive, but I could see it going either way.